src/data/CONTRIBUTORS.ts

```diff
@@ -17,6 +17,15 @@ const CONTRIBUTORS = ensureRecord<Contributor>()({
 	// 		JOBS.SOME_JOB,
 	// 	],
 	// },
+	ACKWELL: {
+		name: 'ackwell',
+		avatar: 'https://secure.gravatar.com/avatar/383e78034167fdfe8303231a6dad33e5?s=100',
```

Comment:
i can't believe it took me 5 years to work out that i could use a gravatar for this shit